test/e2e/framework/aws/eventbridge/eventbridge.go

```diff
@@ -56,7 +56,7 @@ func ForceDeleteEventBus(ebClient eventbridgeiface.EventBridgeAPI, eventBusName
 // any associated rule should also be deleted.
 func deleteEventBus(ebClient eventbridgeiface.EventBridgeAPI, eventBusName string, force bool) {
 	if force {
-		removeAllRules(ebClient, eventBusName)
+		deleteAllRules(ebClient, eventBusName)
```

Comment:
I renamed this helper so that it better matches the name of the actual API call, which is `DeleteRule` (see screenshot above).